src/index.html

```diff
@@ -53,7 +53,13 @@
         <script src="scripts/index.ts" type="module"></script>
     </head>
     <body class="dark-mode">
-        <header class="header"></header>
+        <header class="header">
+            <h1 class="">FairShare</h1>
+            <div class="btn-controller">
+                <button id="toggle-btn">Light Mode</button>
```

Comment:
Can we provide more meaningful name here ? 
```suggestion
                <button id="theme-toggle-btn">Light Mode</button>
```

Reply:
add https://github.com/dhruvdhaduk-simform/fair-share/pull/5/commits/c6c1fbb58e9420bf4bec7ce1e6dc9f296ca2c9dc